src/golang/cmd/server/handler/get_config.go

```diff
@@ -0,0 +1,26 @@
+package handler
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/aqueducthq/aqueduct/config"
+)
+
+type GetConfigHandler struct {
+	GetHandler
+}
+
+func (*GetConfigHandler) Name() string {
+	return "GetConfig"
+}
+
+func (h *GetConfigHandler) Prepare(r *http.Request) (interface{}, int, error) {
```

Comment:
This is basically a no-op. I'm not sure what's the best way to skip or just ignore what comes from here in golang. Or is the better way to do things to do the work of getting the config here and then pass as a parameter into Perform?